package.json

```diff
@@ -19,43 +19,34 @@
     "node": ">=4.8.4"
   },
   "scripts": {
-    "test": "grunt test"
+    "clean": "rm -rf package-lock.json && rm -rf node_modules && rm -rf coverage",
+    "test": "mocha --recursive 'test/**/*.js' --reporter spec --timeout 3000 --ui bdd --exit",
+    "test:watch": "npm run test -- -w ./lib",
+    "lint": "jshint lib/**/*.js test/**/*.js --config .jshintrc",
+    "test:coverage": "istanbul cover _mocha -- --recursive 'test/**/*.js' --reporter spec --exit",
+    "watch": "watch 'npm test && npm run lint' ./lib ./test"
   },
   "devDependencies": {
```

Comment:
As done in PR https://github.com/telefonicaid/iotagent-node-lib/pull/682, this PR should include a similar description of dependencies upgrade and removed dependencies in CHANGES_NEXT_RELEASE file

Reply:
Fixed with ca5f3959de06ad0a2bd8bdd6485d236169fa98d3